src/QsCompiler/LanguageServer/EditorState.cs

```diff
@@ -261,11 +261,15 @@ void AddProperty(IDictionary<string, string?> props, string property, params str
                     return;
                 }
 
-                var newManager = CompilationUnitManager.InitializeFileManager(textDocument.Uri, textDocument.Text, this.publish, ex =>
+                bool isNotebook = textDocument.LanguageId.Contains("notebook");
```

Comment:
I feel like this string should be a constant, but not sure where to put it. Also, I'm not sure if this check is robust enough. I'll send y'all the accompanying code on the Azure Notebooks side (it's very simple)

Reply:
Even though we don't check the language ID anywhere else, I think I'd still feel more comfortable with a strict equality check when we do check it.